src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorTool.java

```diff
@@ -57,6 +57,13 @@ public static void main(String[] args) throws Exception {
             .hasArg()
             .argName("OUTPUT-DIRECTORY")
             .build());
+    options.addOption(
```

Comment:
Yeah, I agree

I admittedly did it this way to save time... but I'll be sure to do file based configuration in the refactoring